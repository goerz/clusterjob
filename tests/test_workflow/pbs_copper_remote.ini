[Attributes]
remote = copper
backend = pbs
shell = /bin/bash
rootdir = /work/goerz
workdir = cjtest
ssh = /usr/local/ossh/bin/ssh
scp = /usr/local/ossh/bin/scp

[Resources]

queue = debug
time = 00:05:00
select = 1:ncpus=32:mpiprocs=32
-A = XXXXXXXXXXXXX
-j = oe
