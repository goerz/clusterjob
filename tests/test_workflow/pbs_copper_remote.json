[
  {
    "args": [
      [
        "mkdir",
        "-p",
        "/work/goerz/cjtest"
      ],
      "copper"
    ],
    "kwargs": {
      "ignore_exit_code": false,
      "ssh": "/usr/local/ossh/bin/ssh"
    },
    "response": "\n\n"
  },
  {
    "args": [
      [
        "qsub",
        "test_clj.pbs"
      ],
      "copper",
      "/work/goerz",
      "cjtest"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "/usr/local/ossh/bin/ssh"
    },
    "response": "\n\n436041.sdb\n"
  },
  {
    "args": [
      [
        "qstat",
        "-x",
        "436041"
      ],
      "copper"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "/usr/local/ossh/bin/ssh"
    },
    "response": "\n\nJob id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----\n436041.sdb        test_clj         goerz             00:00:01 R debug           \n"
  },
  {
    "args": [
      [
        "qdel",
        "436041"
      ],
      "copper"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "/usr/local/ossh/bin/ssh"
    },
    "response": "\n\n"
  },
  {
    "args": [
      [
        "mkdir",
        "-p",
        "/work/goerz/cjtest"
      ],
      "copper"
    ],
    "kwargs": {
      "ignore_exit_code": false,
      "ssh": "/usr/local/ossh/bin/ssh"
    },
    "response": "\n\n"
  },
  {
    "args": [
      [
        "qsub",
        "test_clj.pbs"
      ],
      "copper",
      "/work/goerz",
      "cjtest"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "/usr/local/ossh/bin/ssh"
    },
    "response": "\n\n436042.sdb\n"
  },
  {
    "args": [
      [
        "qstat",
        "-x",
        "436042"
      ],
      "copper"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "/usr/local/ossh/bin/ssh"
    },
    "response": "\n\nJob id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----\n436042.sdb        test_clj         goerz             00:00:01 R debug           \n"
  },
  {
    "args": [
      [
        "qstat",
        "-x",
        "436042"
      ],
      "copper"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "/usr/local/ossh/bin/ssh"
    },
    "response": "\n\nJob id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----\n436042.sdb        test_clj         goerz             00:00:01 R debug           \n"
  },
  {
    "args": [
      [
        "qstat",
        "-x",
        "436042"
      ],
      "copper"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "/usr/local/ossh/bin/ssh"
    },
    "response": "\n\nJob id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----\n436042.sdb        test_clj         goerz             00:00:01 R debug           \n"
  },
  {
    "args": [
      [
        "qstat",
        "-x",
        "436042"
      ],
      "copper"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "/usr/local/ossh/bin/ssh"
    },
    "response": "\n\nJob id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----\n436042.sdb        test_clj         goerz             00:00:01 R debug           \n"
  },
  {
    "args": [
      [
        "qstat",
        "-x",
        "436042"
      ],
      "copper"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "/usr/local/ossh/bin/ssh"
    },
    "response": "\n\nJob id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----\n436042.sdb        test_clj         goerz             00:00:01 R debug           \n"
  },
  {
    "args": [
      [
        "qstat",
        "-x",
        "436042"
      ],
      "copper"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "/usr/local/ossh/bin/ssh"
    },
    "response": "\n\nJob id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----\n436042.sdb        test_clj         goerz             00:00:01 F debug           \n"
  }
]
