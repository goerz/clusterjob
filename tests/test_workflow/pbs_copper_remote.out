####################################################
Job id         : 436042.sdb
Job name       : test_clj
Workdir        : /work/goerz/cjtest
Submission Host: copper02
Compute Node   : nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
Job started on copper07 Sun Feb 28 16:24:31 CST 2016
Current directory: /lustre/usr/local/u/goerz
####################################################
Computer node:
nid00916
Application 2665968 resources: utime ~0s, stime ~1s, Rss ~4032, inblocks ~24, outblocks ~28
Job Finished:  Sun Feb 28 16:25:32 CST 2016
